demo-app/src/main/res/xml/backup_rules.xml

```diff
@@ -1,13 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?><!--
-   Sample backup rules file; uncomment and customize as necessary.
-   See https://developer.android.com/guide/topics/data/autobackup
-   for details.
-   Note: This file is ignored for devices older that API 31
-   See https://developer.android.com/about/versions/12/backup-restore
--->
+<?xml version="1.0" encoding="utf-8"?>
 <full-backup-content>
-    <!--
-   <include domain="sharedpref" path="."/>
-   <exclude domain="sharedpref" path="device.xml"/>
--->
+    <exclude
+        domain="sharedpref"
+        path="__androidx_security_crypto_encrypted_file_pref__.xml" />
+    <exclude
+        domain="file"
+        path="datastore/quick-editor-preferences.preferences_pb" />
```

Comment:
This is what we would have to add to the documentation to make sure third-party devs include those rules in their config. 

We can provide this in the library, but there's no guarantee they will be used. If the third-party app has its own rules, ours will be discarded. 

With the extra code to clean up the backup files, we could technically avoid this. But I would rather now have those file at all to not do the cleanup at all. The `__androidx_security_crypto_encrypted_file_pref__` file is not used only by us.

Reply:
I think we should provide these rules as a default configuration within the SDK if we can, so that it works out of the box for most use cases.

Reply:
>I think we should provide these rules as a default configuration within the SDK if we can, so that it works out of the box for most use cases.

Android Studio adds empty rules by default to a new project. We have to assume that the configuration we provided won't work as a default case.